source/stylesheets/components/_hover_animation.scss

```diff
@@ -0,0 +1,4 @@
+.HoverAnimation {
```

Comment:
Can you provide documentation for this? Can you please state there why was this needed as well?